### PR TITLE
fix the return value of the function

### PR DESCRIPTION
**Pull request by deepoil**
Hi,
This PR is Fixed the following:

1. NewBot, Delete and Flush function seemed not to return error.
 (Always only nil is entered in error)
**→ Remove return error**

2. In the Get function,
 there was a place where if was judged for both value and bool, 
but I thought it would be okay to judge only bool.
**→ Change to bool only judgment**

3. I fixed the part that seems to be a typo.
*I'm sorry if it is not what I expected.
**stringified → string field**

Please check when you have time.

### CODE REVIEW

**Comment by deepoil** [comment]
@oklahomer 
Thank you for confirming.
I understand that UserContextStorage.Delete() and UserContextStorage.Flush() may return an error.
I'll fix it and re-commit.

**Comment by deepoil** [comment]
@oklahomer 
I restored the following. (All relevant parts)
・UserContextStorage.Delete() and UserContextStorage.Flush()
・Stringified

Please check when you have time.

File: examples/status/main.go [hunk 1]
```diff
@@ -1,6 +1,5 @@
 /*
 Package main provides an example that uses sarah.CurrentStatus() to get current go-sarah and its belonging Bot's status via HTTP server.
-
```
Comment (oklahomer) [inline examples/status/main.go:3]: Irrelevant change?

**Comment by oklahomer** [comment]
@deepoil Thank you for the update. Now your changes include those that I considered appropriate 👍 

However, the change to `sarah.NewBot()` includes a breaking change that modifies the method signature. As I suggested in my previous comment, a major version update is required for such a change. Please take a look at my previous suggestion about `v4`. The other change is a tiny internal fix that can be applied at any moment, but as the breaking change is included in this pull request, I cannot proceed to merge.

This kind of difficulty typically occurs when a single pull request contains multiple changes/fixes.

**Comment by deepoil** [comment]
> sarah.NewBot()

File: examples/status/main.go (hunk 1 above)
Comment (deepoil) [reply examples/status/main.go:3]: Yes, I'm sorry my mistake.

**Comment by oklahomer** [comment]
@deepoil Here comes the [v4-dev](https://github.com/oklahomer/go-sarah/tree/v4-dev) branch. You can create a pull request for `sarah.NewBot()`.

**Pull request closed without merging**